forms-flow-bpm/forms-flow-bpm-camunda/src/main/java/org/camunda/bpm/extension/keycloak/rest/KeycloakAuthenticationFilter.java

```diff
@@ -38,7 +38,7 @@ public class KeycloakAuthenticationFilter implements Filter {
 	/** This class' logger. */
 	private static final Logger LOG = LoggerFactory.getLogger(KeycloakAuthenticationFilter.class);
 	
-	private static List<String> HARD_CODED_ROLES = Arrays.asList("create_designs", "view_designs", "create_submissions", "view_submissions", "view_tasks", "manage_tasks", "admin", "resubmit_submissions");
```

Comment:
Hi @auslin-aot are we removing `resubmit_submissions` permission / role?

Reply:
This permission was added for the new resubmission page, but was never used, so I removed it. @arun-s-aot